lib/Email/Stuffer.pm

```diff
@@ -180,8 +180,9 @@ Creates a new, empty, Email::Stuffer object.
 =cut
 
 sub new {
-	my $class = ref $_[0] || $_[0];
-
+	my $proto = shift;
+	my $class = ref $proto || $proto;
```

Comment:
I don't understand the purpose of these two lines to replace the existing line.  $proto is defined and then thrown away.